pages/feedback.js

```diff
@@ -20,22 +20,21 @@ export default class extends Component {
       basePath: '/feedback',
       ssoToken: null
     });
-  }
+  });
```

Comment:
This should happen on the first mount only, right? If so, we should add `[]` as a dependency to `useEffect`.